learning-development/learning-support.qmd

```diff
@@ -10,10 +10,15 @@ As analysts and statistics producers, we require a variety of tools to efficient
 
 For best practice when using software and coding in our process, see our [guidance on RAP expectations](../RAP/rap-expectations.html) and the [DfE Good Code Practice guide](https://dfe-analytical-services.github.io/good-code-practice/index.html).
```

Comment:
Should we add something acknowledging that this is all focussed on technical support and for any general questions about statistics process or practices people can ask HoP office?

Reply:
Have added a sentence to cover this but it might need rewording

Reply:
Looks good - now I've seen it I feel left out, so how about a third paragraph of this?

`For support in data harmonisation and data structuring standards, or support with analytical digital services including the explore education statistics service, and deploying and maintaining R Shiny applications contact the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk).`